app/models/status.rb

```diff
@@ -79,6 +79,17 @@ class Status < ApplicationRecord
   scope :including_silenced_accounts, -> { left_outer_joins(:account).where(accounts: { silenced: true }) }
   scope :not_excluded_by_account, ->(account) { where.not(account_id: account.excluded_from_timeline_account_ids) }
   scope :not_domain_blocked_by_account, ->(account) { account.excluded_from_timeline_domains.blank? ? left_outer_joins(:account) : left_outer_joins(:account).where('accounts.domain IS NULL OR accounts.domain NOT IN (?)', account.excluded_from_timeline_domains) }
+  scope :tagged_with_all, ->(tags) {
```

Comment:
I guess stylistically if you need a multiline lambda for a scope, the scope is better off being a class method

Reply:
There's a subtle difference here in that in a class method, `self` refers to the class itself (ie, Status, which would map to Status.all), whereas within a scope `self` refers to the existing relation that the scope's being called on.

So a straight port to a class method doesn't quite work because instead of reducing the existing relation, we start reducing from all statuses